src/utils/style-formatter.js

```diff
@@ -35,7 +35,7 @@ const setFontsize = (textElement, newFontsize, fontFamily) => {
 
 export default {
   getStyles({ style, disabled, Theme, element }) {
-    let styles = 'width: 100%;height: 100%;padding: 4px;';
+    let styles = 'width: 100%;height: 100%;padding: 4px;transition: transform .1s ease-in-out;';
```

Comment:
Did we check with UX what speed it should have?

Reply:
I showed Christofer a video of it and seemed fine with it. He probalby wants to play around with it to make sure